AddDefenderExceptions.ps1

```diff
@@ -33,24 +33,41 @@ if ($WindowsPrincipal.IsInRole($AdminRole)) {
 	exit
 }
 
-write-host "Important: Run this script from the directory root with which Rookie will be run to exclude"
-Start-Sleep -s 5
+write-host "Run this script from the directory root with which Rookie will be run"
+start-sleep -s 5
+
+$paths = @(
```

Comment:
Please remove \Rookie from the paths. Since $PSScriptRoot already resolves to C:/RSL/Rookie in this context, appending \Rookie results in a duplicated path (C:/RSL/Rookie/Rookie). This would prevent the exclusions from functioning as intended.